redaxo/src/addons/install/lib/api_package_download.php

```diff
@@ -22,7 +22,7 @@ public function execute()
         $packages = rex_install_packages::$function();
         $this->fileId = rex_request('file', 'int');
         if (!isset($packages[$this->addonkey]['files'][$this->fileId])) {
-            return null;
+            throw new rex_api_exception('The requested addon version can not be loaded, maybe it is already installed.');
```

Comment:
Die version noch in die error msg rein

Reply:
Ich hab an dieser Stelle keine Version, nur eine interne ID, die mMn. nicht weiterhilft. Die Version dazu liegt ja gerade nicht vor.